command/proto/command.proto

```diff
@@ -24,6 +24,7 @@ message Request {
 	bool transaction = 1;
 	repeated Statement statements = 2;
 	int64 dbTimeout = 3;
+	map<string, string> metadata = 4;
```

Comment:
So why do this? I am wondering if I had use context objects properly and throughout rqlite (I started coding rqlite long before contexts were introduced) would this be necessary?